bool fix_principal_point
bool fix_aspect_ratio
bool zero_tangent_dist
uint16 num_coeffs
float64[] param_ranges
float64 alpha
---
sensor_msgs/CameraInfo result
---
CalibrationCoverage feedback


